charts/team-ns/templates/external-secrets.yaml

```diff
@@ -3,49 +3,80 @@
 {{- if get (index $v.apps "external-secrets") "enabled" }}
 {{- range $s := $v.secrets }}
 {{- $ns := index $s "namespace" | default $.Release.Namespace }}
-{{- $pathPrefix := printf "/secret/data/teams/team-%s" $v.teamId }}
+{{- $pathPrefix := printf "secret/data/teams/team-%s" $v.teamId }}
 {{- $vaultPath := printf "%s/%s" $pathPrefix $s.name | quote }}
-apiVersion: kubernetes-client.io/v1
+{{- $teamNamespacesCollection := print "team-" (include "helm-toolkit.utils.joinListWithSep" (dict "list" $v.teamIds "sep" ",team-")) }}
+
+apiVersion: external-secrets.io/v1beta1
+{{- if and (eq $v.teamId "admin") (or ($s.clusterWide | default false) ($s.teamWide | default false)) }}
```

Comment:
Instead of constantly checking a a teamId is admin. Would it b e possible to simplify logic

```
{{ if teamId == admin }}
apiVersion: external-secrets.io/v1beta1
kind: ExternalSecret | ClusterExternalSecret
{{ else }}
apiVersion: external-secrets.io/v1beta1
kind: ExternalSecret 
{{end}}

```

Reply:
It seems the same to me and your proposal is less DRY. Can we move on?